check_pgactivity

```diff
@@ -3351,6 +3355,100 @@ sub check_btree_bloat {
 }
 
 
+=item B<session_stats> (14+)
+
+Gather miscellaneous session statistics.
+
+This service uses the status file (see --status-file parameter).
+
+Perfdata contains the session / active / idle-in-transaction times
+for each database since last call, as well as the number of sessions per second,
+and the number of sessions killed / abandoned / terminated by fatal errors.
+
+Required privileges: unprivileged role.
+
+=cut
+
+sub check_session_stats {
+    my @rs;
+    my @perfdata;
+    my @hosts;
+    my %sstats;
+    my %new_sstats;
+    my $instance_session_rate;
+    my %args  = %{ $_[0] };
+    my $me    = 'POSTGRES_SESSION_STATS';
+    my $sql   = q{
+        SELECT extract(EPOCH from now()), s.datname, s.sessions,
+            s.session_time, s.active_time, s.idle_in_transaction_time,
+	    s.sessions_killed, s.sessions_abandoned, s.sessions_fatal
+        FROM pg_stat_database s
+          JOIN pg_database d ON s.datid = d.oid
+        WHERE d.datallowconn
+    };
+
+    @hosts = @{ parse_hosts %args };
+
+    pod2usage(
+        -message => 'FATAL: you must give only one host with service "session_stats".',
+        -exitval => 127
+    ) if @hosts != 1;
+
+    is_compat $hosts[0], 'session_stats', $PG_VERSION_140 or exit 1;
+
+    %sstats = %{ load( $hosts[0], 'session_stats', $args{'status-file'} ) || {} };
+
+    @rs = @{ query( $hosts[0], $sql ) };
+
+    $new_sstats{$_->[1]} = {
+        'ts'               => $_->[0],
+        'sessions'         => $_->[2],
+        'session_time'     => $_->[3],
+        'active_time'      => $_->[4],
+        'idle_in_tx_time'  => $_->[5],
```

Comment:
nitpicking: everywhere else in the code we use xact for transaction, not tx, so it would be better to be consistent here.